dashboard/app/models/pd/application/teacher1819_application.rb

```diff
@@ -451,5 +451,37 @@ def meets_criteria
     def total_score
       response_scores_hash.values.map {|x| x.try(:to_i)}.compact.reduce(:+)
     end
+
+    # @override
+    # Filter out extraneous answers based on selected program (course)
+    def self.filtered_labels(course)
+      labels_to_remove = (course == 'csd' ?
+        [:csp_which_grades, :csp_course_hours_per_week, :csp_course_hours_per_year, :csp_terms_per_year, :csp_how_offer, :csp_ap_exam]
```

Comment:
nit: this line is wider than our [max](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#all-languages) 120 chars. Please split up one symbol per line for readability. Same below.